awscli/data/codebuild/2016-10-06/completions-1.sdk-extras.json

```diff
@@ -0,0 +1,102 @@
+{
+  "merge": {
+    "resources": {
+      "Projects": {
```

Comment:
Let's make this singular (e.g. `Project`). Typically, the resource definition are singular.